modules/understanding-upgrade-channels.adoc

```diff
@@ -82,6 +83,27 @@ and pre-production and production environments that participate in connected cus
 You can use the `stable-{product-version}` channel to upgrade from a previous minor version of
 {product-title}.
 
+[discrete]
+== eus-{product-version} channel
+
+In addition to the stable channel, certain minor versions of {product-title} offer an
+Extended Update Support[https://access.redhat.com/support/policy/updates/openshift#ocp4_phases] (EUS).
+These EUS versions extend the maintenance phase for customers with Premium Subscriptions
+to 14 months. {product-title} {product-version} is a minor version that includes EUS.
+
+Although there is no difference between stable-{product-version} and eus-{product-version}
+channels until {product-version} transitions to EUS phase, you can switch
```

Comment:
"to EUS phase" -> "to the EUS phase"?

Reply:
how about simply "to EUS, you may switch..." ?

Reply:
nevermind, I'll go with "to the EUS phase..."